website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,144 @@
+Docker containers are available on [Docker Hub](https://hub.docker.com/r/hashicorp/consul)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+1. Take a snapshot:
+1. Inspect the snapshot to ensure that Consul successfully captured it using the following command:
+this snapshot somewhere safe. Refer to the following documentation for additional information about creating and using snapshots:
+
+    - [Consul Snapshot](/consul/commands/snapshot)
+    - [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+
+1. Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
+is set to `debug`. After doing this, issue the following command on your servers to
+reload the configuration:
+1. Remove references to services in non-default namespaces and partitions from service resolver, service splitter, and service router configuration entries.
+
+
+## Perform the Downgrade
+
+**1.** Issue the following command to discover which server is currently the leader:
+
+```
+consul operator raft list-peers
+```
+
+You should receive output similar to this (exact formatting and content may differ based on version):
+
+```
+Node       ID                                    Address         State     Voter  RaftProtocol
+dc1-node1  ae15858f-7f5f-4dcb-b7d5-710fdcdd2745  10.11.0.2:8300  leader    true   3
+dc1-node2  20e6be1b-f1cb-4aab-929f-f7d2d43d9a96  10.11.0.3:8300  follower  true   3
+dc1-node3  658c343b-8769-431f-a71a-236f9dbb17b3  10.11.0.4:8300  follower  true   3
+```
+Take note of which agent is the leader.
+
+**2.** Restart or redeploy all clients with a CE version of the binary. You can use a service management system (e.g., systemd, upstart, etc.) to restart the Consul service. If you are not using a service management system, you must restart the  agent manually.
+
+**3.** Update server binaries to use the CE version
+
+**4.** The following steps must be done in order on the server agents, leaving the leader
+agent for last. First, an env var named `CONSUL_ENTERPRISE_DOWNGRADE_TO_CE` should be set to `true` on the servers. Then use a service management system (e.g., systemd, upstart, etc.) to restart the Consul service. If you are not using a service management system, you must restart the agent manually.
+
+To validate that the agent has rejoined the cluster and is in sync with the leader, issue the
+following command:
+
+```
+consul info
+```
+
+Check whether the `commit_index` and `last_log_index` fields have the same value. If done properly,
+this should avoid an unexpected leadership election due to loss of quorum.
+
+**5.** Double-check that all servers are showing up in the cluster as expected and are on
+the correct version by issuing:
+
+```
+consul members
+```
+
+You should receive output similar to this:
+
+```
+Node       Address         Status  Type    Build  Protocol  DC
+dc1-node1  10.11.0.2:8301  alive   server  1.8.3  2         dc1
```

Comment:
It is better we fix the build number in the example output, as using an older version would make the reader think that this functionality is supported in older versions as well.

Reply:
have modified the build to 1.18 since we are releasing this feature in 1.18